src/screens/profile/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { MainButton, Text } from 'src/components';
+import { AppColor, ButtonMode } from 'src/common/enums';
+import { BUTTON_PROFILE_NAVBAR_WIDTH } from 'src/common/constants';
+import styles from './styles';
+
+type State = Array<{ id: number; text: string }>;
+
+interface Props {
+  active: number;
+  handleClick: (a: number) => void;
+}
+
+const Navbar = ({ active, handleClick }: Props) => {
+  const scrollRef = useRef<ScrollView>(null);
+
+  useEffect(() => {
+    const scrollToPosition = active * BUTTON_PROFILE_NAVBAR_WIDTH;
+    scrollRef.current?.scrollTo({
+      x: scrollToPosition,
+      y: 0,
+      animated: true,
+    });
+  }, [active]);
+
+  const [items] = useState<State>([
+    { id: 0, text: 'Summary' },
+    {
+      id: 1,
+      text: 'Qualities',
+    },
+    {
+      id: 2,
+      text: 'Interests',
+    },
+    {
+      id: 3,
+      text: 'Skills',
+    },
+    {
+      id: 4,
+      text: 'Education',
+    },
+  ]);
+
+  return (
+    <View style={styles.navbar}>
+      <Text style={styles.title}>User profile</Text>
+      <View style={styles.buttons}>
+        <ScrollView horizontal={true} ref={scrollRef}>
+          {items.map((item, index) => (
+            <MainButton
+              key={item.id}
+              style={active === index ? styles.active : styles.btn}
```

Comment:
```suggestion
              style={[styles.button, active && styles.buttonActive]}
```

Reply:
It works but like this: `style={[styles.button,  activeIndex === index && styles.buttonActive ] }`